src/main/MySweetProgram.java

```diff
@@ -0,0 +1,37 @@
+package main;
+
+public class MySweetProgram {
+    public static void main (String[] args) {
+        Car Car = new Car();
+        Car.setMaker("Honda");
+        Car.setModel("Acura");
+        Car.setCountryOfCar("Japan");
+        System.out.println(Car.getCountryOfCar() + " " + Car.getModel() +  " " + Car.getMaker());
+    }
+}
+class Car {
+    public String countryOfCar;
+    public String Model;
+    private String Maker;
+    public String Output() {
+        return (getCountryOfCar()+getMaker() + getModel());
+    }
+    public void setCountryOfCar(String Japan) {
```

Comment:
please use 'camalCase' for all variables names and use abstract names for variables like "countryOfCar" instead of "japan".